suse_migration_services/units/mount_system.py

```diff
@@ -128,15 +128,27 @@ def mount_system(root_path, fstab):
             mountpoint = ''.join(
                 [root_path, fstab_entry.mountpoint]
             )
+            device = fstab_entry.device
+            if not os.path.exists(device):
+                # device doesn't exist in the migration root, it might be
+                # a file on the host that we want to loopback mount...
+                device = ''.join([root_path, device])
+                if not os.path.exists(device):
+                    # ...but if that *also* doesn't exist, log a warning and
+                    # skip this entry.
+                    log.warning('Not mounting {0} ({1} does not exist)'.format(
+                        mountpoint, fstab_entry.device))
+                    continue
+
```

Comment:
I'm OK with it. I am just not convinced of the use case and the value added of picking up every mount point from the root system. For migration we really only need the mounted devices that represent the system as a whole according to FHS.

Reply:
> For migration we really only need the mounted devices that represent the system as a whole 

correct, and that's why we take all information from /etc/fstab into account.

The patch here is ok but to be honest I don't see this to be a responsible of the mount_system service. As Robert already mentioned only system relevant mount points should be in fstab. Everything else does not belong here. So the reason why this patch was created was because of an fstab that lists iso files to be mounted that are not relevant for the system. So from my point of view a misuse of the fstab and I think that should not lead to code we have to maintain here.